File: fix-imports.py
import os
import re

dir_path = os.path.join('src', 'common', 'components', 'ui', 'promptUi')
print(f"Checking directory: {dir_path}")

fixed_count = 0

for filename in os.listdir(dir_path):
    if not (filename.endswith('.tsx') or filename.endswith('.ts')):
        continue
    
    filepath = os.path.join(dir_path, filename)
    print(f"Processing {filepath}")
    
    with open(filepath, 'r', encoding='utf-8') as file:
        content = file.read()
    
    if '@/lib/utils' in content:
        new_content = content.replace('@/lib/utils', '../../../../libs/utils')
        
        with open(filepath, 'w', encoding='utf-8') as file:
            file.write(new_content)
        
        fixed_count += 1
        print(f"Fixed import in {filename}")

print(f"\nFixed imports in {fixed_count} files.") 

File: fix-imports.js
const fs = require('fs');
const path = require('path');

const directory = path.join(__dirname, 'src', 'common', 'components', 'ui', 'promptUi');
console.log(`Looking in directory: ${directory}`);

try {
  const files = fs.readdirSync(directory);
  console.log(`Found ${files.length} files`);

  let fixedCount = 0;

  files.forEach(file => {
    if (!file.endsWith('.tsx') && !file.endsWith('.ts')) return;
    
    const filePath = path.join(directory, file);
    try {
      let content = fs.readFileSync(filePath, 'utf8');
      
      if (content.includes('import { cn } from "@/lib/utils"')) {
        const newContent = content.replace(
          'import { cn } from "@/lib/utils"',
          'import { cn } from "../../../../../libs/utils"'
        );
        
        fs.writeFileSync(filePath, newContent, 'utf8');
        fixedCount++;
        console.log(`Fixed import in ${file}`);
      }
    } catch (err) {
      console.error(`Error processing file ${file}:`, err);
    }
  });

  console.log(`\nFixed imports in ${fixedCount} files.`);
} catch (err) {
  console.error('Error:', err);
} 